packages/api/src/task/scheduler.ts

```diff
@@ -539,10 +539,44 @@ export class TaskScheduler {
       if (typeof asset === "string") {
         asset = await db.asset.get(asset);
       }
+
+      let phase = asset.status?.phase;
+      // Prevent bump of updatedAt if phase isn't getting updated
+      let updatedAt = asset.status?.updatedAt;
+
+      if (phase == "ready") {
```

Comment:
You gotta remember to use triple equals in JS, double equal can do weird stuff sometimes. As a rule of thumb, **always** use triple equals unless you have a reason not to.
```suggestion
      if (phase === "ready") {
```